README.md

```diff
@@ -35,6 +36,21 @@ cmake -G "Visual Studio 16 2019" -A x64 -T v141_xp -DORC_BUILD_VCPKG=ON ..
 cmake --build . --config MinSizeRel -- -maxcpucount
 ```
 
+For Visual Studio 2017:
+```bash
+git clone https://github.com/dfir-orc/dfir-orc.git
+cd dfir-orc
+mkdir build-x86 build-x64
+
+cd build-x86
+cmake -G "Visual Studio 15 2017" -A Win32 -T v141_xp -DORC_BUILD_VCPKG=ON ..
```

Comment:
The correct command line should not include -A x64.

Reply:
Thank you for the correction